### PR TITLE
completes task 06-get-comment-by-article-id

### PR DESCRIPTION
**Pull request by ellsbee**
completes task 06-get-comment-by-article-id and updated endpioints.JSON file

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -124,4 +123,43 @@ describe('testing correct access to /api/articles', () => {
...
+                comment_id: expect.any(Number), 
+                votes: expect.any(Number), 
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String), 
+                article_id: expect.any(Number), 
+                }) 
+            ); 
+        }); 
+    });
```
Comment (MRombi) [inline __tests__/app.test.js:145]: We could add a test where we expect an empty array for when an article exists but there are no comments under it 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -124,4 +123,43 @@ describe('testing correct access to /api/articles', () => {
     })
 });
 
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200 - returns all associated comments from a specific article when searched for by article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => { 
+        expect(body.comments).toBeInstanceOf(Array);
```
Comment (MRombi) [inline __tests__/app.test.js:132]: We do not need that this is an instance of array but we should check the array length to avoid false positives when asserting inside a for each 🙂

**Review by MRombi — COMMENTED** [review]
Please have a look at my comments 🙂
